scripts/dashboard/build-dashboard.ts

```diff
@@ -1,22 +1,48 @@
-const { writeFile } = require('fs-extra');
-const { resolve } = require('path');
-const { graphql } = require('@octokit/graphql');
-const { Queries } = require('./issue-queries');
-
-/**
- * Introduces a delay in the execution flow.
- * @param {number} ms - The number of milliseconds to pause.
- * @returns {Promise<void>} A promise that resolves after the specified delay.
- */
-async function pause(ms) {
+import { graphql } from '@octokit/graphql';
+import assert from 'assert';
+import { writeFile } from 'fs-extra';
```

Comment:
Add the types of this library as devDependency in package.json

Reply:
I have searched online but couldn't find types for @octokit/graphql library. So I have declared necessary types manually in the `/types` directory. 

Other libraries such as assert, fs-extra takes type definitions from `@types/node` which We have already installed.